posts/2021/10/numpy_benchmarking_blog.md

```diff
@@ -0,0 +1,79 @@
+<!--
+.. title: NumPy Benchmarking
+.. slug: numpy-benchmarking
+.. date: 2021-09-27 12:23:40 UTC-05:00
+.. author: Khushi Agrawal
+.. tags: NumPy, Pythran, Numba, Transonic
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+.. previewimage:
+-->
+
+In this blog post, I'll be talking about my journey in Quansight. I want to share all things I was involved in and accomplished. What issues I faced, and most importantly, what were awesome life hacks I learned during this period.
+
+First of all, I'd like to express my gratitude to the whole team for allowing me to be a part of such a great team. My work was majorly focused on providing performance benchmarks to NumPy in realistic situations. The target was to show the world that NumPy is efficient in handling quasi real-life situations too.
+
+<p align="center">
+      <img src = "/images/2021/10/journey.jpeg" alt = "A word cloud with themes, open-source projects and people mentioned throughout the blog post. Each is stylized using a different font, most of them calligraphical.">
+</p>
+
+## My Experience
+My work was broadly divided majorly into the following categories:
+
+- **N-Body Problem**: The [N-Body problem](https://en.wikipedia.org/wiki/N-body_problem) is one of the most famous and universally accepted problems for benchmarking. I was given this as a problem statement to work on the project. I started my work with a theoretical understanding of the problem. It was a fun learning part for me to connect the scientific part with the programming world. I implemented the N-Body problem in Python, C, and C++.
```

Comment:
Please give credit for the original code with a link to the repo. Something like "I adopted the code from ..."

Reply:
Yes, sure!